dev/zsh-completion.py

```diff
@@ -33,13 +33,14 @@ def generate_completion(opt):
         completion += ":_files"
     elif opt.name in ["plugins_url", "motd_url"]:
         completion += ":_urls"
-    helpstr = opt.helpstr.replace("[", "\\[").replace("]", "\\]")
```

Comment:
Thanks for pushing a fix so quickly! =)

Can you explain the change?

Reply:
just conceal `[]`
```zsh
    --fmt_expand_list'[format str to use for names of columns expanded from list (colname, index) (default: %s\[%s\])]:str' \
```

Reply:
I see, the option default value in line 42 has to be escaped also, and now one of our default values has `[`/`]` in it.